index.js

```diff
@@ -72,8 +72,8 @@ module.exports = function (zipPath, opts, cb) {
 
         // if no mode then default to readable
         if (mode === 0) {
-          if (isDir) mode = 365 // 0555
-          else mode = 292 // 0444
+          if (isDir) mode = parseInt(opts.defaultDirMode,8) || 493 // Default to 0755
```

Comment:
I'd prefer it if you could do the options parsing on a separate line to prevent undefined behavior. I like to have each line do one thing, even if it means having 5 lines instead of 1 line. Also this line causes the JS style check to fail (which is why travis failed on the PR). You can run `npm test` to run the tests and you should see the error in your console